<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1476</width>
    <height>1081</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Трансформации</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Модель</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QComboBox" name="modelsCB"/>
              </item>
              <item>
               <widget class="QPushButton" name="loadModelBtn">
                <property name="text">
                 <string>Загрузить</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="deleteModelBtn">
                  <property name="text">
                   <string>Удалить</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="deleteModelsBtn">
                  <property name="text">
                   <string>Удалить все</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>Перемещение</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="1" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Ox</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Oy</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="7">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QPushButton" name="moveBtn">
                  <property name="text">
                   <string>Переместить</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="moveAllBtn">
                  <property name="text">
                   <string>Переместить все</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Oz</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="6">
               <widget class="QDoubleSpinBox" name="dxDSB">
                <property name="minimum">
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>65536.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="6">
               <widget class="QDoubleSpinBox" name="dyDSB">
                <property name="minimum">
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>65536.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="6">
               <widget class="QDoubleSpinBox" name="dzDSB">
                <property name="minimum">
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>65536.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_8">
             <property name="title">
              <string>Поворот</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="2" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Oy</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Ox</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Oz</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="6">
               <widget class="QDoubleSpinBox" name="axDSB">
                <property name="minimum">
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>65536.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="6">
               <widget class="QDoubleSpinBox" name="ayDSB">
                <property name="minimum">
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>65536.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="7">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QPushButton" name="rotateBtn">
                  <property name="text">
                   <string>Повернуть</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="rotateAllBtn">
                  <property name="text">
                   <string>Повернуть все</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="1" colspan="6">
               <widget class="QDoubleSpinBox" name="azDSB">
                <property name="minimum">
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>65536.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_9">
             <property name="title">
              <string>Масштабирование</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="1" column="0">
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>Ox</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>Oy</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="7">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QPushButton" name="scaleBtn">
                  <property name="text">
                   <string>Масштабировать</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="scaleAllBtn">
                  <property name="text">
                   <string>Масштабировать все</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>Oz</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="6">
               <widget class="QDoubleSpinBox" name="kxDSB">
                <property name="minimum">
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>65536.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="6">
               <widget class="QDoubleSpinBox" name="kyDSB">
                <property name="minimum">
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>65536.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="6">
               <widget class="QDoubleSpinBox" name="kzDSB">
                <property name="minimum">
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>65536.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Камера</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QComboBox" name="cameraCB"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
             <item>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QPushButton" name="topLeftBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>⭯</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="upBtn">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>🡱</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="topRightBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>⭮</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="leftBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>🡰</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="botLeftBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>↩</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="downBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>🡳</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QPushButton" name="botRightBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>↪</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="rightBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>🡲</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QPushButton" name="addCameraBtn">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Добавить</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteCameraBtn">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Удалить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1476</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
